api/swagger.yaml

```diff
@@ -833,6 +858,73 @@ paths:
             $ref: '#/responses/Success'
         default:
           $ref: '#/responses/UnknownError'
+
+  /control/gateways:
+    get:
+      summary: 'List of configured gateways'
+      responses:
+        '200':
+          description: 'Success'
+          schema:
+            type: 'object'
+            required:
+              - gateways
+            properties:
+              gateways:
+                type: 'array'
+                description: 'List of gateways'
+                items:
+                  $ref: '#/definitions/APIGateway'
+        default:
+          $ref: '#/responses/UnknownError'
+    post:
+      summary: 'Create a new gateway'
+      parameters:
+        - in: 'body'
+          name: 'body'
+          required: true
+          schema:
+            description: 'Create a new gateway'
+            $ref: '#/definitions/APIGateway'
```

Comment:
Can the client provide a Gateway with ID and token, or those always generated by APIClarity? 
In that case, shall we create an object like APIGateway_Create? I am not sure what's the common practise in APIclarity today

Reply:
The client can, it will be validated, but it will be ignored when creating the entry.
So, APIClarity generates the ID and the Token regardless.